.coveragerc

```diff
@@ -598,6 +598,7 @@ omit =
     homeassistant/components/mochad/*
     homeassistant/components/modbus/climate.py
     homeassistant/components/modbus/cover.py
+    homeassistant/components/modbus/fan.py
```

Comment:
Again it would be nice if we could avoid having to exclude it.

Reply:
I plan to...